libs/ui/src/components/MediaItem/MediaItem.vue

```diff
@@ -83,6 +85,7 @@ const props = withDefaults(
     src: '',
     animationSrc: '',
     mimeType: '',
+    sizes: '225px md:175px lg:135px',
```

Comment:
Is it compatible with other types of media item?

Reply:
Yes other types ignore this prop

Reply:
Then we should move this default value inside `libs/ui/src/components/MediaItem/type/ImageMedia.vue`